### ISSUE TITLE
Hover tekst som forklarer knappene

### ISSUE THREAD

**Issue by helemork** [Artsdatabanken/nin-kart-frontend#1473]
Forslag fra @Myrdrahl om hover tekst som forklarer knappene

**Comment by helemork** [comment]
### Map function
Describe all buttons in the map interface to screen readers

A lot of the buttons in the map control area do have descriptions to screen readers. But some of them lack a description and some of them do have descriptions that isn’t explanatory enough for users to understand what they actually do.

[her var en haug med irrelevante eksempler som ikke lengre eksisterer]

![bilde](https://user-images.githubusercontent.com/17450081/65227711-6d630b80-dac9-11e9-8409-1da42e1042ee.png)

- [ ] Velg kartlag” doesn’t say what layer this is, or what will happen if I click on the button. In this case, the layer is already displayed, so its very hard to understand what will happen.
- [ ] Also, we recommend using aria-label rather than the title attribute on buttons. Title are used for an extra explanation, not as an alternative, while the purpose with aria-label is to provide a label on the button. [H: Her har han misforstått. Denne er ment som det den er - ekstrashjelp for vanlig bruker, ikke alternnativ visning. Å legge til aria kommer i tillegg!]
### 
These buttons don’t explain what data they affect: 
![bilde](https://user-images.githubusercontent.com/17450081/65227811-98e5f600-dac9-11e9-8eb6-e6e16441cbd0.png)

They say “Show/hide” and “pick colour”, but they don’t say that one is for “ikke isbredominert” and the other is for “Isbredominert”. 
- [ ]  Dump inn navn i forklaringen

### One more example:
![bilde](https://user-images.githubusercontent.com/17450081/65227897-c16df000-dac9-11e9-8e12-6c3744e9922d.png)
The plus icons have a description, but it doesn’t include the name of the layer. The arrow icons have no descriptions at all which is very unlucky since that are links to text pages which would be of extra interest to visually impaired users.
Even if most users who can’t see wouldn’t use the map there are situations where a blind user would want to show a map to a colleague or a friend. Since the interface is here, it should be explained.

### 
Suggested solution
We recommend:
•	Use aria-label rather than the title attribute to describe buttons.
•	All buttons should not only describe the function, but also what data they affect.
### Priority (1-4)
High (2) 
### WCAG relation
1.3.1 Info and relationships (level A)
4.1.2 Name, Role, Value (level A)
